src/applications/check-in/components/ExternalLink.jsx

```diff
@@ -13,6 +13,8 @@ function ExternalLink({
   eventId = null,
   eventPrefix = '',
   dataTestId = 'external-link',
+  target = null,
+  rel = null,
```

Comment:
I made this change in what I am currently working on too. Do these null values case default behavior though? This 
is how I did it: https://github.com/department-of-veterans-affairs/vets-website/commit/8068fa88a4d3ee4a5f43804f32379117ce3f3f92

Reply:
I think so here is a link without those props passed in `<a href="/health-care/how-to-file-a-va-travel-reimbursement-claim-online/" hreflang="en" data-testid="find-out-link">Find out how to file for travel reimbursement</a>`

and with `<a href="http://va.gov/accessva-travel-claim" hreflang="en" target="_blank" rel="noreferrer" data-testid="sign-in-btsss-link">Sign in to BTSSS to check your claim status</a>`

Reply:
oh nice, guess I could have "tested" that 😵

Reply:
The other thing, is that you need to account for app like I did in the commit I shared up there. Otherwise it counts all clicks as `day-of` Honestly you could just wait for this change with my updates. But we will just need to remember to go back and hit this links with it.